tests/post_training/model_scope.py

```diff
@@ -6,7 +6,7 @@
 def get_validation_scope() -> List[dict]:
     model_scope = []
     # Basic
-    model_scope.append({"name": "vgg11", "quantization_params": {}})
+    #model_scope.append({"name": "vgg11", "quantization_params": {}})
```

Comment:
Commenting the code is not good. But I would like to keep these models to get back in the future and for ramping up quickly when doing local experiments.